revscoring/languages/tests/test_dutch.py

```diff
@@ -0,0 +1,123 @@
+    "porn", "porno", "neuk", "neuke", "neuken", "neukende", "neukt", "neukte",                             
+    "suck", "sucks", "suckt", "zuigt", "sukkel", "sukkels", "tering", "tetten",                             
+    "verkracht", "dikzak", "dildo", "mogolen", "mogool", "mongool", "mooiboy", "neger",                             
+    "kut", "kutje", "kutjes", "stelletje", "loser", "losers", "lul", "lullen",                             
+    "reet", "scheet", "scheten", "schijt", "diaree", "slet", "lekkerding", "likken",   
+    "utme", "utml", "utmn", "utmz", "rdn", "cest", "cet", "sophonpanich",
+    "boe", "dombo", "domme", "godverdomme", "izan", "kots", "kusjes", "lekker", "lekkere",
+    "lkkr", "nerd", "nerds", "noob", "noobs", "sex", "sexy", "stink", "stinken", "stinkt",
+    "stoer", "vies", "vieze", "vuile", "xxx", "zielig", "zooi", "swag", "swek", "yolo"
+]
+
+INFORMAL = [
+    "hoi", "hey", "hallo", "doei", "heej", "heey", "groetjes", "halloo", "hoihoi", "hoii",
+    "hoiii", "heb", "zeg", "vind", "bent", "snap", "boeit", "klopt", "hou", "houd",
+    "gwn", "maarja", "nou", "ofzo", "oke", "yeah", "hoor", "hihi", "eigelijk", "heeel", "jij",
+    "jou", "jullie", "sorry", "vetgedrukte", "deelonderwerp", "cursieve", "nowiki", "bewerkingsveld",
+    "cursief", "minecraft", "hotmail", "hyves", "bieber", "werkstuk", "spreekbeurt", "haha", "hahah",
+    "hahaha",     "hahahah", "hahahaha", "hahahahah", "hahahahaha", "hahahahahaha", "lala", "lalala",
+    "lalalala", "lalalalala", "bla", "blabla", "blablabla", "cool", "coole", "coolste", "dikke", "dom",
+    "lelijk", "lelijke", "lelijkste", "leuk", "leuke", "lief", "onzin", "raar", "saai", "gek", "gekke",
+    "goeie", "grappig", "stom", "stomme", "filmfocus", "filmliefde", "wikipedianl", "diakrieten", "egt",
+    "jah", "jaja", "jwz", "lol", "lolz", "omg", "tog", "wtf", "zever"
+]
+
+OTHER = [
+    """
+    The Smyth Report is the common name of an administrative history written
+    by physicist Henry DeWolf Smyth about the Manhattan Project, the Allied
+    effort to develop atomic bombs during World War II. It was released to the
+    public on August 12, 1945, just days after the atomic bombings of Hiroshima
+    and Nagasaki. Smyth was commissioned to write the report by Major General
+    Leslie Groves, the director of the Manhattan Project. The Smyth Report was
+    the first official account of the development of the atomic bombs and the
+    basic physical processes behind them. Since anything in the declassified
+    Smyth Report could be discussed openly, it focused heavily on basic nuclear
+    physics and other information which was either already widely known in the
+    scientific community or easily deducible by a competent scientist. It
+    omitted details about chemistry, metallurgy, and ordnance, ultimately
+    giving a false impression that the Manhattan Project was all about physics.
```

Comment:
This block of text should be in dutch.  I recommend copy-pasting from an article on the dutch wiki.